mgmtsystem/hawkular.py

```diff
@@ -128,8 +128,15 @@ def __init__(self,
         self.password = kwargs.get('password', '')
         self.auth = self.username, self.password
         self.inv_api = ContainerClient(hostname, self.auth, protocol, port, "hawkular/inventory")
```

Comment:
Well you would.....it was an if/else, I was just trying to cut down on object instantiation